content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md

```diff
@@ -1,40 +1,76 @@
-Using `do.sh run` we ran the prebuilt images from [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
+Using `./run*.sh` we ran the prebuilt images from [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
 Let's now also build `app-elfloader`.
```

Comment:
Argue why this would be useful. Typically, it won't be; users just care about running applications. But, in case you just did a fix to Unikraft, and you want an `elfloader` image with that fix, you need to build it.